lib/browser/api-util.ts

```diff
@@ -13,4 +13,14 @@ Zone.__load_patch('util', (global: any, Zone: ZoneType, api: _ZonePrivate) => {
   api.patchMethod = patchMethod;
   api.bindArguments = bindArguments;
   api.patchMacroTask = patchMacroTask;
+  // load blackListEvents from global
```

Comment:
you might want to clarify in the comment that the `BLACK_LISTED_EVENTS` name is deprecated, but still supported.

Reply:
Sure, got it!

Reply:
Can you update this plz?

Reply:
Sure, I just updated the comments, please review, thanks!